sdk/healthinsights/ci.yml

```diff
@@ -30,13 +30,6 @@ extends:
     ServiceDirectory: healthinsights
     TestProxy: true
     Artifacts:
-    - name: azure-healthinsights-cancerprofiling
-      safeName: azurehealthinsightscancerprofiling
-      skipVerifyChangeLog: true
-
-    - name: azure-healthinsights-clinicalmatching
-      safeName: azurehealthinsightsclinicalmatching
-      skipVerifyChangeLog: true
```

Comment:
Can you open an issue to undo this? we should still be testing these packages

Reply:
@asaflevi-ms these lines were taken out of the ci.yml file because the projects azure-healthinsights-cancerprofiling and azure-healthinsights-clinicalmatching don't build properly can you have look please?

Reply:
Will create github issue to code owner to undo.